content/sensu-go/5.2/_index.md

```diff
@@ -89,3 +89,6 @@ permissive [MIT License][12] and publicly available on [GitHub][13].
 [25]: dashboard/overview
 [26]: sensuctl/reference
 [27]: reference/handlers
+[28]: https://bonsai.sensu.io/assets/sensu/sensu-pagerduty-handler
+[29]: https://bonsai.sensu.io/assets/sensu/sensu-aws
```

Comment:
```suggestion
[29]: https://bonsai.sensu.io/assets/sensu/sensu-influxdb-handler
```

Reply:
That's not a valid link, but we can update this to reference the Prometheus collector instead.